utils/config.ts

```diff
@@ -9,13 +9,18 @@ import * as t from "io-ts";
 import { ValidationError } from "io-ts";
 import { readableReport } from "@pagopa/ts-commons/lib/reporters";
 import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { UrlFromString } from "@pagopa/ts-commons/lib/url";
 
 // global app configuration
 export type IConfig = t.TypeOf<typeof IConfig>;
 // eslint-disable-next-line @typescript-eslint/ban-types
 export const IConfig = t.interface({
+  AppQueueStorageConnection: NonEmptyString,
+  DGC_BASE_URL: UrlFromString,
```

Comment:
we should have a different url for each environment (prod, uat and load)

Reply:
Fixed in f0c089a